packages/appstore/src/helpers/total-assets-helper.ts

```diff
@@ -0,0 +1,34 @@
+import { useCFDAccounts, useExchangeRate, usePlatformAccounts } from '@deriv/hooks';
+
+export const isRatesLoaded = (
+    is_real: boolean,
```

Comment:
Do we need to pass is_real ?
Can we get it from the hooks

Reply:
@thisyahlen-deriv it's just a helper pure function, not React component